openmicroscopy/bioformats
# Maven examples 2

Follow-up of https://github.com/openmicroscopy/bioformats/pull/2535

This PR migrates the remaining Java example files from `docs/sphinx/developers` to `docs/sphinx/developers/examples` and adds execution configuration to the Maven `pom.xml`.

To test this PR:
- check all Java examples are executed in the output of the `mvn` command and that files are created under `docs/sphinx/target`
- check the documentation build still passes.
